notes/grh-non-hrf-postpay.vbs

```diff
@@ -518,6 +520,7 @@ DO
 			IF err_msg <> "" AND ButtonPressed = -1 THEN MsgBox "*** NOTICE!!! ***" & vbCr & err_msg & vbCr & vbCr & "Please resolve for the script to continue."
 		LOOP UNTIL err_msg = "" OR ButtonPressed = previous_button
 	LOOP WHILE ButtonPressed = previous_button
+	IF worker_signature = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
```

Comment:
worker signature is outside of the do...loop with the err_msg. Update to within the err_msg do....loop.

Reply:
was going to add 
    MAXIS_footer_month = right("00" & MAXIS_footer_month, 2)
    MAXIS_footer_year = right("00" & MAXIS_footer_year, 2)
    MAXIS_footer_month_confirmation	'function will check the MAXIS panel footer month/year vs. the footer month/year in the dialog, and will navigate to the dialog month/year if they do not match.

but I didnt want it to be out of scope.. and then I had a question so I left it alone